mava/utils/make_env.py

```diff
@@ -56,34 +57,31 @@
     "Cleaner-v0": {"generator": CleanerRandomGenerator, "wrapper": CleanerWrapper},
 }
 
-# Define a different registry for Matrax since it has no generator.
+# Define a different registry for other suites since they have no generator.
```

Comment:
One final suggestion on this, could we rename the `_jaxmarl_wrappers -> _jaxmarl_registry` like the others and maybe change the comment to:
 `# Registry mapping environment names directly to the corresponding wrapper classes.`